server/domain/status/Status.js

```diff
@@ -96,6 +96,8 @@ class Status {
     static createStatus(data) {
         console.log('[Status] Creating status...');
 
+        this.formatUserImage(data);
```

Comment:
Would rather see `data = this.filterStatus(data)` here

Reply:
Yes, as discussed I am introducing a statusFactory that does all the heavy lifting for us :)

Reply:
Done!